Extensions/xkit_patches.js

```diff
@@ -664,10 +664,17 @@ XKit.tools.dump_config = function(){
 					if ($(".html-field").css("display") === "none") {
 						var content_editor = $('.post-form--form').find('.editor.editor-richtext');
 						if (content_editor.length === 0) {
-							XKit.console.add('ERROR: unable to set content html');
+							XKit.console.add('ERROR: unable to get content html');
 							return '';
 						}
 						return content_editor.html();
+					} else {
+						var html_or_markdown = $(".tab-label[data-js-srclabel]").text();
+						if (html_or_markdown === 'HTML') {
+							return $('.ace_editor .ace_text-layer').text();
+						}
+						XKit.window.show('Invalid editor selection', 'ERROR: XKit cannot currently get content from the markdown editor. Click <a href="https://www.tumblr.com/settings/dashboard">here</a> to edit your dashboard settings to use the rich text or HTML editors.', 'error', "<div id=\"xkit-close-message\" class=\"xkit-button\">OK</div>");
+						return '';
```

Comment:
still worried that this leaves ER erasing the entire content of reblogged posts. Would be best to just bail out of ER altogether. (with a try/catch maybe?)